criu/config.c

```diff
@@ -420,6 +423,31 @@ static int parse_join_ns(const char *ptr)
 	return 0;
 }
 
+static int parse_file_validation_method(struct cr_options *opts, const char *optarg)
+{
+	if (!strcmp(optarg, "filesize")) {
+		opts->file_validation_method = FILE_VALIDATION_FILE_SIZE;
```

Comment:
you have to describe all these options and their values in the usage message (crtools.c) and in Documentation/criu.txt